MIT.md

```diff
@@ -0,0 +1,8 @@
+## Copyright 2021, [Ant&oacute;nio Jos&eacute; Gomes](https://www.linkedin.com/in/ajffg1024)
+
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this **PROJECT** and associated documentation files, to deal in the **PROJECT** without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the **PROJECT**, and to permit persons to whom the **PROJECT** is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the **PROJECT**.
+
+THE **PROJECT** IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE **PROJECT** OR THE USE OR OTHER DEALINGS IN THE [APP TYPE].
```

Comment:
This is the generic MIT license file but not edited yet to this specific project.

Wherever **PROJECT** is found, it SHOULD be replaced by the name of this project.

Reply:
- [ ] It is important when giving reviews to phrase your comments politely by using polite suggestions such as `I think it would be better to use semantic tags`, `I want to kindly suggest`, `please can you make this necessary change` etc

Reply:
I think that it would be better to replace the **PROJECT** word by the name of the present project, like this, for example:

```suggestion
# MIT License

## Copyright (c) 2023, [Ant&oacute;nio Jos&eacute; Gomes](https://www.linkedin.com/in/ajffg1024)


Permission is hereby granted, free of charge, to any person obtaining a copy of this **project-template project** and associated documentation files, to deal in the **project-template project** without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the **project-template project**, and to permit persons to whom the **project-template project** is furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all copies or substantial portions of the **project-template project**.

THE **project-template project** IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE **project-template project** OR THE USE OR OTHER DEALINGS IN THE SITE.
```